src/components/Footer/Footer.tsx

```diff
@@ -1,10 +1,7 @@
+import styles from './Footer.module.scss';
+
 const Footer = () => {
-  return (
-    <header>
-      <h1>Footer</h1>
-      {/* 추후 푸터 내용 추가 */}
-    </header>
-  );
+  return <footer className={styles.footer}>© 2025 GᖇOᗯᒪOG. All rights reserved.</footer>;
```

Comment:
```suggestion
  const fullYear = new Date().getFullYear();

  return <footer className={styles.footer}>{`© ${fullYear} GᖇOᗯᒪOG. All rights reserved.`}</footer>;
```

이렇게 사용하면 년도가 바껴도 수정안해도 됩니다!